client/my-sites/site-settings/settings-security/main.jsx

```diff
@@ -46,9 +49,20 @@ const SiteSettingsSecurity = ( {
 		);
 	}
 
+	const credentials = find( rewindState.credentials, { role: 'main' } );
+	const isManaged = credentials && credentials.type && 'managed' === credentials.type;
+
+	const isRewindActive = [ 'awaitingCredentials', 'provisioning', 'active' ].includes(
+		rewindState.state
+	);
+	const hasScanProduct = sitePurchases.some( p => p.productSlug.includes( 'jetpack_scan' ) );
```

Comment:
Should we check if the scan product is available though a plan as well? Or would that be cough by the isRewindActive check?

Reply:
In my testing, plans are currently covered by the isRewindActive check since Scan isn't ever bundled without Backup. This could totally change in the future, though. I agree it's something we should revisit, likely with some API involvement.